near/omni-bridge/src/btc.rs

```diff
@@ -0,0 +1,177 @@
+use crate::storage::NEP141_DEPOSIT;
+use crate::{
+    ext_token, Contract, ContractExt, Role, FT_TRANSFER_CALL_GAS, ONE_YOCTO, STORAGE_DEPOSIT_GAS,
+};
+use near_plugins::{access_control_any, pause, AccessControllable, Pausable};
+use near_sdk::json_types::U128;
+use near_sdk::{
+    env, near, require, serde_json, AccountId, Gas, Promise, PromiseError, PromiseOrValue,
+};
+use omni_types::btc::{TokenReceiverMessage, UTXOChainConfig};
+use omni_types::{ChainKind, Fee, OmniAddress, TransferId, TransferMessage};
+
+const SUBMIT_TRANSFER_TO_BTC_CONNECTOR_CALLBACK_GAS: Gas = Gas::from_tgas(5);
+
+#[derive(Debug, near_sdk::serde::Deserialize)]
+struct UTXOChainMsg {
+    version: u8,
```

Comment:
I propose to use enum instead of version field. From may perspective the field version isn't useful a lot here because you still have to deserialize the msg somehow

Reply:
fixed: https://github.com/Near-One/omni-bridge/pull/348/commits/7de11c101757f1938b50bcb87603cb5f6fed56e0

Reply:
@frolvanya can you please verify this change, I have to make sure that this kind of json format is convenient for sdk and indexers